pubspec.yaml

```
@@ -20,7 +20,7 @@ dev_dependencies:
   flutter_lints: ^5.0.0
 
 # The following section is specific to Flutter packages.
-flutter: 
+flutter:
   # Add assets if needed
   assets:
-    - assets/
+  - assets/
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)